src/gui/gui.pri

```diff
@@ -55,6 +55,7 @@ HEADERS += \
     $$PWD/tagfilterproxymodel.h \
     $$PWD/tagfilterwidget.h \
     $$PWD/banlistoptions.h \
+    $$PWD/ipsubnetwhitelistoptionsdialog.h \
```

Comment:
Everything seems settled down, so could you, please, make the same changes to src/gui/CMakeLists.txt?

Reply:
Updated CMakeLists.txt (and squashed).